api/app/parsers/encuentra_parser.js

```diff
@@ -10,14 +12,37 @@ exports.parse = function(html) {
   return $list.map(function() {
             var $item = $(this),
                 options = {
-                  title: $item.find('.ann-box-title').text(),
-                  price: ($item.find('.ann-price-2nd').text()).replace('B/.', ''),
-                  sold : ($item.css('opacity')) ? true : false,
-                  image: $item.find('.lazy').attr('data-original') ? ($item.find('.lazy').attr('data-original')).replace('/small','') : null,
-                  link : encuentraConfig.baseUrl() + $item.find('.ann-box-title').attr('href'),
-                  app  : encuentraConfig.app()
-
+                  title    : $item.find('.ann-box-title').text(),
+                  price    : ($item.find('.ann-price-2nd').text()).replace('B/.', ''),
+                  sold     : ($item.css('opacity')) ? true : false,
+                  image    : $item.find('.lazy').attr('data-original') ? ($item.find('.lazy').attr('data-original')).replace('/small','') : null,
+                  link     : encuentraConfig.baseUrl() + $item.find('.ann-box-title').attr('href'),
+                  createdAt: parseCreatedAt($item.find('.ann-box-hilight-time > span.value').text()),
+                  app      : encuentraConfig.app()
                 };
             return standardizer.do(options);
           }).get();
 }
+
+// Parses created at text to a timestamp date
+function parseCreatedAt(createdAtText) {
```

Comment:
Change method name to ``convertCreatedAtTextToTimestamp`` to better communicate intent.